lovely/shaders.toml

```diff
@@ -0,0 +1,36 @@
+[manifest]
+version = "1.0.0"
+dump_lua = true
+priority = -10
+
+# Shader patching
+[[patches]]
+[patches.pattern]
+target = "game.lua"
+pattern = '''            self.SHADERS[shader_name] = love.graphics.newShader("resources/shaders/"..filename)'''
+position = "at"
+payload = '''
+local shader = "resources/shaders/"..filename
+local lovely = require"lovely"
+if lovely.apply_patches then
+    local res, err = lovely.apply_patches(filename, love.filesystem.read("resources/shaders/"..filename))
+    if res then
+        shader = res
+    else
+        sendErrorMessage(err, 'ShaderPatcher')
```

Comment:
It may be worth the error case just throwing (assert the call to apply_patches). I don't really think it's expected behaviour to silently use the original file if lovely fails (in lua buffers this would panic)